packages/vitest/src/integrations/env/happy-dom.ts

```diff
@@ -5,9 +5,12 @@ import { populateGlobal } from './utils'
 export default <Environment>({
   name: 'happy-dom',
   transformMode: 'web',
-  async setupVM() {
+  async setupVM({ 'happy-dom': happyDom = {} }) {
```

Comment:
I think naming it `happyDom` in the config is fine

Reply:
The types can be found here:
```javascript
     import IHappyDOMOptions from 'happy-dom/lib/window/IHappyDOMOptions.js';
```